### PR TITLE
LPS-166241 [EVP] Create Page Organization Detail

### PR DESCRIPTION
**Pull request by juliocezar96**
[LPS-166241](https://issues.liferay.com/browse/LPS-166241)

### CODE REVIEW

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]
LGTM :heavy_check_mark:

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by juliocezar96** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/03_company/page-definition.json [hunk 1]
```diff
@@ -283,10 +283,10 @@
 												"borderRadius": "",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/03_company/page-definition.json:283]: Please rename this folder to `03_organization` and any other "company" to "organization"

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @juliocezar96

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Pull request closed without merging**